src/main/java/org/lwes/emitter/MulticastEventEmitter.java

```diff
@@ -44,27 +44,15 @@
  * @author Anthony Molinaro
  * @since 0.0.1
  */
-public class MulticastEventEmitter extends AbstractEventEmitter {
+public class MulticastEventEmitter extends DatagramSocketEventEmitter<MulticastSocket> {
 
   private static transient Log log = LogFactory.getLog(MulticastEventEmitter.class);
 
-  /* the actual multicast socket being used */
-  private MulticastSocket socket = null;
-
-  /* the multicast address */
-  private InetAddress address = null;
-
-  /* the multicast port */
-  private int port = 9191;
-
   /* the multicast interface */
-  private InetAddress iface = null;
+  protected InetAddress iface = null;
```

Comment:
address and iface are actually different, address is the address of the host you will emit the messages to, iface is the network interface (in case you have more than one) on the host that would be used to emit the messages.